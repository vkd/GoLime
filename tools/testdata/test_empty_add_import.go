package test

func unusedEmpty() string {
	return "unused_empty"
}
