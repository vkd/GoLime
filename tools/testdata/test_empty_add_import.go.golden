package main

import "bytes"

func main() {

}
